lib/main_common.pm

```diff
@@ -1428,6 +1428,8 @@ sub load_extra_tests {
     loadtest "console/hostname";
     if (any_desktop_is_applicable()) {
         load_extra_tests_desktop;
+        # both load console test and x11 test in one job for migration test
+        load_extra_tests_textmode if (check_var('EXTRATEST', 'upgrade'));
```

Comment:
looks really strange to load console test under condition  DESKTOP !~ 'textmode' ... I understand that we doing select_console 'root-console' anyway , but still . Something wrong here ... but probably not in this PR but generally in our concept of DESKTOP variable . Just thinking loud , don't take it as NO to merge

Reply:
@asmorodskyi  I agree with you.  I supporse it would be better to use `EXTRATEST` variable in the condition part. I would like try to load those case to migration job group with minimal impact for the functional group in the meantime.
@okurz do you have any idea about it?

Reply:
Well, it is good intention to not brake functional tests but I would prefer a clean solution though. In the end, the *only* thing you want to change is the schedule and that can be more easily tested, e.g. download the vars.json files from the extra tests, also the maintenance ones, and check the schedule by executing just `isotovideo -d` in a newly created directory where you put the vars.json file to see how the schedule evaluates.

Reply:
since we have all upgrade cases with gnome/kde desktop installed that can easily perform `load_extra_tests_desktop`, what about to simply create a textmode 12sp3 image to upgrade to sle15 and then perform `load_extra_tests_textmode` that also a benefit to see how the upgrade process in textmode, in this case we could follow the existing schedule and not break any other functional tests.